docs/calibration.md

```diff
@@ -1,8 +1,32 @@
 ## Calibrated URDF for manipulation planning
 
-By default we load an URDF from `assets/hab_stretch/urdf/stretch_manip_mode.urdf` for manipulation. You need to replace this with the appropriate URDF for your robot.
+Calibration is crucial for good grasping performance on the real robot, and will also improve performance of heuristic methods for navigation.
 
-This adds a couple dummy joints to make 6dof planning easier, but isn't going to be calibrated for your specific robot. If you follow the calibration instructions from Hello, you can copy your `stretch.urdf` to your desktop though, and modify it to add these dummy joints.
+  1. Copy the DexWrist URDF and xacro onto your robot from the [Stretch tool share](https://github.com/hello-robot/stretch_tool_share/tree/master/tool_share/stretch_dex_wrist).
+  1. Follow the [calibration instructions](https://github.com/hello-robot/stretch_ros/tree/master/stretch_calibration) provided by Hello Robot.
+  1. Copy your `stretch.urdf` from the robot and update it as below, in "Loading the New URDF in HomeRobot", *if necessary*. This is an advanced tutorial; you should be able to get everything working without it, but might see worse grasping performance on your hardware.
+
+Once you follow the [calibration instructions](https://github.com/hello-robot/stretch_ros/tree/master/stretch_calibration), you will have an URDF on your robot in `$HOME/catkin_ws/src/stretch_description/urdf`. You can check it with this command:
+```
+ls `rospack find stretch_description`/urdf | grep stretch.urdf
+```
+You should see the file listed. Running the installation guide, you should also see a visualization of your robot, having loaded this model.
+
+### Explanation of Terms
+
+*urdf*: Unified Robot Description Format. Basically, an XML file that describes a robot's geometry. Check out the [ROS documentation](https://docs.ros.org/en/foxy/Tutorials/Intermediate/URDF/URDF-Main.html) for some details if you are curious.
+
+*Calibration*: your robot's geometry may vary slightly from ours; while we provide a calibrated URDF together with [our habitat models](https://github.com/cpaxton/hab_stretch), it may not work for you.
```

Comment:
This definition doesnt actually explain the term "calibration" :P

Reply:
updated